src/server/db/mongo-db.js

```diff
@@ -0,0 +1,15 @@
+import mongoose from 'mongoose'
+
+import config from '../server.config'
+
+const { uri, user, pass, dbName } = config.mongodb
+mongoose.Promise = Promise
+mongoose.db = mongoose.createConnection(uri, {
+  user,
+  pass,
+  dbName,
```

Comment:
no need for user,pass,dbname they should be passed through the uri

Reply:
done